### PR TITLE
Commit for Refactoring of code. #11

### PR DESCRIPTION
**Pull request by mihirterna**
Moved common methods into single static helper class, for better reusability. Formatted java classes for better readability. Tested every changes in local. Working fine as expected.

### CODE REVIEW

**Comment by mihirterna** [comment]
Please add label hacktoberfest-accepted for this PR. Thanks.

**Comment by Muhib007dev** [comment]
Can you add one more package of service layer and move the business logic from controller into that package. Also try to refactor the variable names as well.

**Comment by mihirterna** [comment]
Ok, can you please give example on service layer abstraction ?

File: src/main/java/com/coronaapi/coronarestapi/controller/WebController.java [hunk 1]
```diff
@@ -21,80 +17,60 @@
...
-				arrayList.add(new AllCountryModel(table.getCellAt(i, j+1).asText(), table.getCellAt(i, j + 2).asText(),
-						table.getCellAt(i, j + 4).asText(), table.getCellAt(i, j + 6).asText(),
-						table.getCellAt(i, j + 8).asText()));
+		for (int i = 9; i < 224; i++) {
+			for (int j = 0; j < 1; j++) {
+				arrayList.add(new AllCountryModel(table.getCellAt(i, j + 1).asText(),
+						table.getCellAt(i, j + 2).asText(), table.getCellAt(i, j + 4).asText(),
+						table.getCellAt(i, j + 6).asText(), table.getCellAt(i, j + 8).asText()));
 			}
 		}
```
Comment (Muhib007dev) [inline src/main/java/com/coronaapi/coronarestapi/controller/WebController.java]: For example, we can move all this code into the package of the service layer.

**Comment by Muhib007dev** [comment]
In short, I want to make it more readable and manageable, according to the standards.

**Review by Muhib007dev — CHANGES_REQUESTED** [review]

File: src/main/java/com/coronaapi/coronarestapi/controller/WebController.java (hunk 1 above)
Comment (mihirterna) [reply src/main/java/com/coronaapi/coronarestapi/controller/WebController.java]: Thanks for review, Have committed new changes, please review.

Comment (Muhib007dev) [reply src/main/java/com/coronaapi/coronarestapi/controller/WebController.java]: sure. thank you.

**Comment by Muhib007dev** [comment]
Thank you for the contribution.

**Comment by mihirterna** [comment]
Thanks  @Muhib007dev .
Can you please add hacktoberfest-accepted label now, since PR is accepted. Thanks

**Pull request merged**